src/tcgwars/logic/impl/gen3/PopSeries4.groovy

```diff
@@ -241,41 +255,16 @@ public enum PopSeries4 implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 20
+            heal 20, self
           }
         }
       };
       case HEAL_ENERGY_8:
-      return specialEnergy (this, [[C]]) {
-        text "Heal Energy provides Energy. When you attach this card from your hand to 1 of your Pokémon, remove 1 damage counter and all Special Conditions from that Pokémon. If Heal Energy is attached to Pokémon-ex, Heal Energy has no effect other than providing Energy."
-        onPlay {reason->
-        }
-        onRemoveFromPlay {
-        }
-        onMove {to->
-        }
-        allowAttach {to->
-        }
-      };
+      return copy(Deoxys.HEAL_ENERGY_94, this);
       case POKEMON_FAN_CLUB_9:
-      return basicTrainer (this) {
-        text "Search your deck for up to 2 Basic Pokémon and put them onto your Bench. Shuffle your deck afterward."
-        onPlay {
-        }
-        playRequirement{
-        }
-      };
+      return copy(Flashfire.POKEMON_FAN_CLUB_94,this);
```

Comment:
This should copy Aquapolis, or be its own card. The Flashfire version places the basic Pokémon in the hand, this version places them on the bench.